sycl/include/sycl/detail/pi.h

```diff
@@ -74,9 +74,11 @@
 // PI_EXT_ONEAPI_CONTEXT_INFO_USM_MEMSET2D_SUPPORT, and
 // PI_EXT_ONEAPI_CONTEXT_INFO_USM_MEMCPY2D_SUPPORT context info query
 // descriptors.
+// 12.22 Added new piextEnqueueDeviceVariableWrite and
```

Comment:
Tag @kbenzie to note the PI API addition

Reply:
I've created https://github.com/oneapi-src/unified-runtime/issues/150 to match these changes when completed.